backend/src/apis/csv_parser.rs

```diff
@@ -1,49 +1,118 @@
-use std::error::Error;
-use std::fs::File;
-use std::path::Path;
+use std::{error::Error, fs::File, path::Path};
 
+use ethers::{
+    abi::AbiEncode,
+    types::{Bytes, U256},
+};
 use serde::Deserialize;
 
+use crate::contracts::generated::summa_contract::{AddressOwnershipProof, Asset};
+
 #[derive(Debug, Deserialize)]
-struct Record {
+struct EntriesRecord {
```

Comment:
Can we rename it to `SignatureRecord` as the other is named `AssetRecord` ?

Reply:
Updated at [31c3c59](https://github.com/summa-dev/summa-solvency/pull/152/commits/31c3c59bf0d5efa6027b238ee7bb54bceb1b8800)